# The initial joint configuration of the robot.
float64[] initial_configuration

# The goal pose of the end effector in the robot base frame.
geometry_msgs/Pose goal_pose

# The pose of the IFCO container in the robot base frame.
geometry_msgs/Pose ifco_pose
---
bool success
